culturefeed_social/includes/blocks.inc

```diff
@@ -362,12 +362,24 @@ function culturefeed_social_user_list_block_for_activity($object, $activity_type
       break;
   }
 
-  try {
-    $activities = DrupalCultureFeed::searchActivities($query);
+  $cache_prefix = 'culturefeed:social:activities:';
+   
+  // If user isn't logged in try to get activities from cache. 
+  if (!culturefeed_is_culturefeed_user() && ($cache = cache_get($cache_prefix . $id . ':' . $activity_type, 'cache_culturefeed'))) {
```

Comment:
Is this necessary?
Do we need to get activities when a user is not logged in?

Reply:
Jup, ook als anonieme gebruiker kan je op detailpagina's lijstjes zien van mensen die een event geliked hebben